.travis.yml

```diff
@@ -8,11 +8,15 @@ branches:
     - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
 matrix:
   include:
-    - python: 3.5
+    - python: "3.5"
+      env: TOXENV=lint
+    # - python: "3.5"
+    #   env: TOXENV=pylint
```

Comment:
I disabled the `pylint` tests in `travis` for now, since they require more attention and should be handled separately.
The test suite can be run with `tox -e pylint`. That will exclude `missing-docs`.